CHANGELOG.md

```diff
@@ -29,6 +23,13 @@
   `loki_tag` is now `logs_instance_tag`, and `backend: loki` is now
   `backend: logs_instance`.
 
+# v0.17.0 (2021-07-15)
+
+- [FEATURE] Added Kafka Lag exporter integration. (@gaantunes)
+
```

Comment:
Blegh, the kafka changelog must have changed between me pulling it. Slightly different, but would rather have this match the changelog in v0.17.0.